bin/i18n/resources/apps/external_sources.rb

```diff
@@ -14,6 +14,12 @@ def self.sync_in
 
           FileUtils.mkdir_p(I18N_SOURCE_DIR_PATH)
 
+          # Preparing Oceans tutorial
+          # TODO: move the file to the `external-sources` folder instead of `blockly-mooc`
+          labs_dir = CDO.dir(File.join(I18N_SOURCE_DIR, 'blockly-mooc'))
+          FileUtils.mkdir_p(labs_dir)
+          FileUtils.cp(CDO.dir('apps/node_modules/@code-dot-org/ml-activities/i18n/oceans.json'), File.join(labs_dir, 'fish.json'))
```

Comment:
I wonder who decided to switch from oceans to fish? We need to investigate how to refactor filenames without loosing translations.

Reply:
https://github.com/code-dot-org/code-dot-org/blame/78190b48869febecc5d575787ab0eb05bbce78f0/bin/i18n-codeorg/in.sh#L43